headers/types/dungeon_mode/dungeon.h

```diff
@@ -909,7 +909,10 @@ struct dungeon {
     // to be set to action::ACTION_WALK until it hits an obstacle.
     bool leader_running;
     undefined field_0x78c;
-    undefined field_0x78d;
+    // 0x78D: This flag is set by the unused move 0x395 (Reviver).
```

Comment:
Hmm, does this mean that `enum move_id` should go higher than it does currently? I can't remember where I pulled the maximum value from, but I definitely didn't go as high as 0x395.

Reply:
Sorry, my bad, that's decimal lol, hexadecimal ID is 0x18B

Reply:
Actually, I've taken a closer look, and this is not set by move 0x18B but by move 0x189, which is the Identify Orb. The flag is not unused, because the Identify Orb signals to the player any monster that holds an item and is not in the team, and this is done by placing an icon on them. This is such an obscure mechanich that I had no idea it even existed.